packages/actions/score/src/modules/scoreboard/sagas.spec.js

```diff
@@ -0,0 +1,64 @@
+import {expectSaga} from 'redux-saga-test-plan'
+import * as matchers from 'redux-saga-test-plan/matchers'
+import {throwError} from 'redux-saga-test-plan/providers'
+import {select} from 'redux-saga/effects'
+import {rest} from 'tocco-app-extensions'
+
+import * as actions from './actions'
+import * as sagas from './sagas'
+
+describe('score', () => {
+  describe('modules', () => {
+    describe('scoreboard', () => {
+      describe('sagas', () => {
+        describe('fetchData', () => {
+          test('should read Tournament-Data from rest-API', () => {
+            const resultData = [
+              {
+                resultUserBean: {
+                  key: '11947',
+                  entity: 'User',
+                  defaultDisplay: 'Hoff, Lorenz'
+                },
+                points: 12,
+                place: 1
+              }
+            ]
+            const entities = {
```

Comment:
nitpick: I personally would rename this variable to something more meaningful in this context

e.g. `response`